src/renderer/screens/Editor/components/OnlyCustomScreen.js

```diff
@@ -25,77 +25,58 @@ import CardContent from "@mui/material/CardContent";
 import Divider from "@mui/material/Divider";
 import Portal from "@mui/material/Portal";
 import Typography from "@mui/material/Typography";
-import withStyles from "@mui/styles/withStyles";
 
 import Focus from "../../../../api/focus";
 import openURL from "../../../utils/openURL";
 import i18n from "../../../i18n";
 import { navigate } from "../../../routerHistory";
 
-const styles = (theme) => ({
-  root: {
-    display: "flex",
-    justifyContent: "center",
-  },
-  card: {
-    margin: theme.spacing(4),
-    maxWidth: "50%",
-  },
-  grow: {
-    flexGrow: 1,
-  },
-});
-
-class OnlyCustomScreen extends React.Component {
-  enableOnlyCustom = async () => {
+const OnlyCustomScreen = (props) => {
+  const enableOnlyCustom = async () => {
     let focus = new Focus();
     await focus.command("keymap.onlyCustom", true);
     await focus.command("settings.defaultLayer", 0);
     await navigate("/editor");
   };
 
-  openFeatureRequest = async () => {
+  const openFeatureRequest = async () => {
     const url =
       "https://github.com/keyboardio/Chrysalis/issues/new?labels=enhancement&template=feature_request.md";
     const opener = openURL(url);
-
     await opener();
   };
 
-  render() {
-    const { classes } = this.props;
-
-    return (
-      <div className={classes.root}>
-        <Portal container={this.props.titleElement}>
-          {i18n.t("app.actionRequired")}
-        </Portal>
-        <Card className={classes.card}>
-          <CardContent>
-            <Typography component="p" gutterBottom>
-              {i18n.t("editor.onlyCustom.warning")}
-            </Typography>
-          </CardContent>
-          <Divider variant="middle" />
-          <CardActions>
-            <Box component="span" mr={1}>
-              <Button onClick={this.openFeatureRequest} variant="outlined">
-                {i18n.t("editor.onlyCustom.openFR")}
-              </Button>
-            </Box>
-            <div className={classes.grow} />
-            <Button
-              onClick={this.enableOnlyCustom}
-              color="primary"
-              variant="outlined"
-            >
-              {i18n.t("editor.onlyCustom.fixItButton")}
+  return (
+    <div sx={{ display: "flex", justifyContent: "center" }}>
+      <Portal container={props.titleElement}>
+        {i18n.t("app.actionRequired")}
+      </Portal>
+      <Card
+        sx={{
+          margin: 4,
```

Comment:
This should probably be `margin: (theme) => theme.spacing(4)`, rather than `margin: 4`.